x/genutil/src/collect_txs.rs

```diff
@@ -19,13 +16,13 @@ pub struct CollectGentxCmd {
     pub moniker: String,
 }
 
-pub fn gen_app_state_from_config<SK: StoreKey>(
+pub fn gen_app_state_from_config(
     CollectGentxCmd {
         gentx_dir,
         home,
         moniker,
```

Comment:
What's the moniker for?

Reply:
[Here](https://github.com/NYBACHOK/gears/blob/89d956062a2910fc7494dd6cfeb0d83feab71f59/x/genutil/src/collect_txs.rs#L140-L143)

Reply:
Cosmos doesn't have that argument?

Reply:
It has

Reply:
[Usage](https://github.com/cosmos/cosmos-sdk/blob/d3f09c222243bb3da3464969f0366330dcb977a8/x/genutil/collect.go#L171-L174) and they read it [here](https://github.com/cosmos/cosmos-sdk/blob/d3f09c222243bb3da3464969f0366330dcb977a8/x/genutil/client/cli/collect.go#L26-L27)